web/src/app/models/youtube.model.ts

```diff
@@ -0,0 +1,9 @@
+export interface Youtube {
+  etag: string;
+  items: Array<string>;
```

Comment:
I think this way of defining an array has been deprecated, and should now be `string[]`, but not a big problem.